src/requirements/Twitter/TwitterRequirement.tsx

```diff
@@ -58,33 +62,77 @@ const TwitterRequirement = (props: RequirementProps) => {
       {...props}
     >
       {(() => {
+        const minCount = Math.floor(requirement.data?.minAmount)
+
         switch (requirement.type) {
           case "TWITTER_NAME":
             return (
               <>
                 <Text as="span">{`Have "`}</Text>
                 <DataBlockWithCopy text={requirement.data.id} />
-                <Text as="span">{`" in your Twitter username`}</Text>
+                <Text as="span">{`" in your X username`}</Text>
               </>
             )
           case "TWITTER_BIO":
             return (
               <>
                 <Text as="span">{`Have "`}</Text>
                 <DataBlockWithCopy text={requirement.data.id} />
-                <Text as="span">{`" in your Twitter bio`}</Text>
+                <Text as="span">{`" in your X bio`}</Text>
               </>
             )
           case "TWITTER_FOLLOWER_COUNT":
-            return `Have at least ${Math.floor(
-              requirement.data.minAmount
-            )} followers on Twitter`
+            return (
+              <Text as="span">
+                {"Have at least "}
+                <DataBlock>{pluralize(minCount, "follower")}</DataBlock>
```

Comment:
I wouldn't put these into DataBlocks, the information is simple enough here to work better as plain text imo